app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -163,7 +163,7 @@ def unlink_from_work
           # Although ActiveFedora clears the children nodes it leaves those fields in Solr populated.
           work.thumbnail = nil if work.thumbnail_id == file_set.id
           work.representative = nil if work.representative_id == file_set.id
-          work.save!
+          persister.save(resource: work)
```

Comment:
Is this just an result of systematically removing `.save!` in favor of the Valkyrie way?  I'm curious as to why not just go the rest of the way and use a `change_set` and a `change_set_persister` over this?  My understanding of the pattern was to keep the persister.save calls into change_set_persisters and prefer saving through them, eg `change_set_persister.save(change_set: change_set)`, whenever possible.  

In Figgy, I'm seeing the pattern for jobs:

https://github.com/pulibrary/figgy/blob/0d6aa5f62237537085e61922eb5e39d5bacd51ca/app/jobs/browse_everything_ingest_job.rb#L5-L15